### PR TITLE
[WIP][JP] Add japanese translation of poolboy docs

### PR DESCRIPTION
**Pull request by iguchi1124**
I'm working on translating poolboy docs into japanese!

### CODE REVIEW

**Comment by riseshia** [comment]
:eyes:

**Comment by doomspork** [comment]
Hi @iguchi1124!  How is your translation coming, are you ready for this to be reviewed?  😁

File: jp/lessons/libraries/poolboy.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+---
+version: 1.1.1
+title: Poolboy
+redirect_from:
+  - /lessons/libraries/poolboy/
```
Comment (eksperimental) [inline jp/lessons/libraries/poolboy.md]: Please remove this redirect

Comment (riseshia) [reply jp/lessons/libraries/poolboy.md]: I will take this pr, so please give me more time ;)

**Pull request closed without merging**